draft-oid4vc-haip-sd-jwt-vc.md

```diff
@@ -103,7 +103,7 @@ Unless explicitly stated, all normative requirements apply to all participating
 Implementations of this profile:
 
 * MUST support both pre-auth code flow and authorization code flow.
```

Comment:
I'd suggest SHOULD support pre-auth code flow and MUST auth code flow because there may be implementers that still have security concerns or implementation constraints about the pre-auth code flow

For the flow that requires a credential offer it would the best solution allowing also a pre auth code flow and not just a pre-authz code flow. This would be an issue for OpenID4VCI, so let's keep for discussion without blocking this PR if possibile

Reply:
there is already issue #60 on this topic